certora/harness/ATokenWithPoolImpl.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+// import {IScaledBalanceToken} from "@aave/core-v3/contracts/interfaces/IScaledBalanceToken.sol";
+import {IATokenWithPool} from "../../contracts/l1/interfaces/IATokenWithPool.sol";
+import {IERC20} from "@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20.sol";
+import {ILendingPool} from "../../contracts/l1/interfaces/ILendingPool.sol";
+import {IAaveIncentivesController} from "../../contracts/l1/interfaces/IAaveIncentivesController.sol";
+import "./DummyERC20ExtendedImpl.sol";
+
+contract ATokenWithPoolImpl is DummyERC20ExtendedImpl {
```

Comment:
Not anymore, since we've replaced it with another implementation of AToken.
This is just for a backup if we want to return to the older version. We might delete it by tomorrow.